VARIABLES.md

```diff
@@ -67,3 +67,5 @@
 | key_name | Hive Metastore EC2 ssh key pair name. | string | automation | no |
 | hms_instance_type | Hive Metastore instance type, possible values ecs, ec2. | string | ecs | no |
 | iam_name_root | Name to identify Hive Metastore IAM roles. | string | hms | no |
+| enable_s3_inventory | Enable S3 inventory configuration. | bool | false | no |
```

Comment:
to be consistent with naming conventions above and to group related configuration could we rename this to `s3_enable_inventory`? And then move it above with the other s3 related variables.